packages/core/src/outcomes.ts

```diff
@@ -280,6 +281,23 @@ export function serverTimeout(msg?: string): OperationOutcome {
   };
 }
 
+export function redirect(url: string): OperationOutcome {
```

Comment:
I wonder if we want to change the signature of this to:
```suggestion
export function redirect(url: URL): OperationOutcome {
```
That way we make sure any caller has to assert this is a valid URL

Just food for thought